src/bbs2gh/Handlers/MigrateRepoCommandHandler.cs

```diff
@@ -344,14 +344,24 @@ private void ValidateOptions(MigrateRepoCommandArgs args)
 
         if (ShouldGenerateArchive(args))
         {
-            if (GetBbsUsername(args).IsNullOrWhiteSpace())
+            if (args.Kerberos)
             {
-                throw new OctoshiftCliException("BBS username must be either set as BBS_USERNAME environment variable or passed as --bbs-username.");
+                if (args.BbsUsername.HasValue() || args.BbsPassword.HasValue())
+                {
+                    throw new OctoshiftCliException("--bbs-username and --bbs-password cannot be provided with --kerberos.");
```

Comment:
Should we thrown and exception or just log that we are ignoring and you no longer need to provide username and password here?

Reply:
I guess if they provide wrong input we need to let them know to fix it.

Reply:
This is similar to my changes when if they provide cloud storage info, we just ignore and log it is not needed for their GHES version. I think you can get product's input on how to handle extra info cc @timrogers.

Reply:
I still think we need to be explicit about garbage/invalid input and so far it's been our approach so even if we want to change it we need to change it everywhere to be consistent. That also means that we probably need to change the cloud storage stuff too if the PR has already been merged. 
@timrogers what do you think?

Reply:
I could go either way. I'd probably lean towards a warning instead of an error, as we can still successfully migrate and ignore extraneous args. Feels like a slightly better UX. But having more strict validations encourages users to do the right things.

Reply:
Sorry for being slow to jump on this thread. I'd prefer to go with the explicit approach where we error if the input params don't make sense together. I realise that's just a subjective preference, but it's the call I'd make 😊